core/controllers/question_editor.py

```diff
@@ -49,15 +64,22 @@ def post(self, skill_id):
         question_dict['question_state_data_schema_version'] = (
             feconf.CURRENT_STATE_SCHEMA_VERSION)
         question_dict['id'] = question_services.get_new_question_id()
+        question_dict['linked_skill_ids'] = skill_ids
```

Comment:
Hmm that's a good point. The reason I am hesitant to do that right now is because @sophiewu6 is also currently working on the frontend related to this feature. I can do that in a follow up, is that fine?

Reply:
Issue has been filed  #6957